pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokerStatsBase.java

```diff
@@ -167,7 +167,7 @@ public LoadManagerReport getLoadReport() throws Exception {
                 throw new RestException(Status.CONFLICT, lm.getClass().getName() + " does not support this operation");
             }
         } catch (Exception e) {
-            log.error("Unable to get Resource Availability - [{}]", e);
+            log.error("Unable to get Resource Availability", e);
```

Comment:
This was done on purpose to avoid having the same stack trace printed twice.

Reply:
Thanks @merlimat . I guess it's should be ``` log.error("Unable to get Resource Availability - [{}]", e.getMessage()); ``` ?

Reply:
Hi @merlimat , I found the other methods are also print stack trace in catch block.

Reply:
we don't want to print out the full trace. we only need to the information of that example. so I think we need to revert this change back.

Reply:
@liketic I think it is case by case. something the exception can be annoying and sometime we need the stacktrace. I would suggest reverting this line of the change. We can let other changes get in first.